### DISCUSSION TITLE
Podman Desktop v0.8.0

### DISCUSSION THREAD

**Discussion by benoitf** [podman-desktop/podman-desktop#494] [Announcements]
Hello,

Podman Desktop has a new release: you can check milestone [v0.8.0](https://github.com/containers/podman-desktop/milestone/8?closed=1)  and [Release Notes](https://github.com/containers/podman-desktop/releases/tag/v0.8.0) 

Downloads are available from the [download section](https://podman-desktop.io/downloads) of the [Podman Desktop website](https://podman-desktop.io/)

Major enhancements are:

- Play kube: when using Podman container engine, deploy pods directly on Podman using the `play YAML/kubernetes` item from the containers list

https://user-images.githubusercontent.com/436777/190343757-bd2786f7-1ca7-4f40-b29b-b9ae05dbbffe.mp4

- Add volumes support
Display the name, creation date and size of the volumes and if they're being used or not

![image](https://user-images.githubusercontent.com/436777/189381923-56417363-d995-478e-a087-f201d5db158a.png)

![image](https://user-images.githubusercontent.com/436777/189381825-7d6ccd91-0243-421f-8cda-b89ffd06b4c1.png)

- Add installation pre-flight checks for macOS

Ensure Podman can be installed/updated properly

<img width="1053" alt="Screenshot 2022-09-01 at 16 18 13" src="https://user-images.githubusercontent.com/929743/187929941-aeb8e926-31bd-49a8-a9a8-5084d93d8528.png">

- Display cpu/memory statistics for a container

https://user-images.githubusercontent.com/436777/189916277-a403ee32-e07e-49da-9fce-27a0c8d6bdd3.mp4

- Fix tray icon on Linux on dark menu bar

![image](https://user-images.githubusercontent.com/436777/191958311-7d79f2a2-4802-4d6b-a2d0-3b997cd826e5.png)

Thanks to @redhatrises for a contribution in https://github.com/containers/podman-desktop/pull/470

Complete list of bug fixes / enhancements https://github.com/containers/podman-desktop/issues?q=is%3Aclosed+milestone%3A0.8.0

**Comment by tazihad** [comment]
will you release in flathub?

**Comment by benoitf** [comment]
@tazihad it is 
https://flathub.org/apps/details/io.podman_desktop.PodmanDesktop

Shows 0.8.0
